rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -39,10 +39,19 @@ namespace rsx
 	{
 		//Don't throw, gather information and ignore broken/garbage commands
 		//TODO: Investigate why these commands are executed at all. (Heap corruption? Alignment padding?)
+		rsx->invalid_command_interrupt_raised |= !(_reg >= 0xac80 >> 2 && _reg < 0xb000 >> 2); // HACK: Allow these "unknown" methods
```

Comment:
This is messy. Just use a simple if...else.
if (in_range) log_trace;
else raise_flag + log error.

Reply:
error logging still occurs for this range, which is why I didn't op for this soluition